backend/app/views/spree/admin/return_index/customer_returns.html.erb

```diff
@@ -0,0 +1,64 @@
+<% content_for :page_title do %>
+  <%= Spree.t(:customer_returns) %>
+<% end %>
+
+<% content_for :table_filter do %>
+  <div data-hook="admin_return_authorisations_index_filters">
+    <%= search_form_for [:admin, @search], url: admin_customer_returns_index_path do |f| %>
```

Comment:
All uses of url helpers should have spree. first to play nice with other engines.  Feel free to update any other instances you notice after rebasing updates from https://github.com/spree/spree/pull/6030